src/components/router/components/Routes.js

```diff
@@ -44,6 +44,13 @@ const Routes = () => {
             <ScrollToTop>
                 <App>
                     <Switch>
+                        {/* <PrivateRoute
+                            authenticated={authenticated}
+                            path={LOCATION}
+                            exact
+                            strict
+                            component={Location}
+                        /> */}
```

Comment:
Are you going to delete this route?

Reply:
@Simpcyclassy  just implemented something  new. Prior before now I was working with that. We still have a discussion concerning that

Reply:
I think I am unaware of a new discussion, I thought I fixed anything that had to do with routes headers and anything that would affect both paths in agreement.